cmpserver/plugin/config.go

```diff
@@ -31,15 +31,20 @@ type PluginConfigSpec struct {
 
 //Discover holds find and fileName
 type Discover struct {
-	Find     Command `json:"find"`
-	FileName string  `json:"fileName"`
+	Find     Find   `json:"find"`
```

Comment:
This e2e test needs to be updated accordingly. https://github.com/argoproj/argo-cd/blob/master/test/e2e/testdata/cmp-find-command/plugin.yaml#L11

Reply:
Forgot to explain this change in PR summary. I moved `glob` field into `Find` struct that inherits `Command`. This is just an internal implementation detail, so end-users or e2e tests are not affected at all.